requirements-dev.in

```diff
@@ -3,11 +3,9 @@ async-solipsism
 async-timeout
 coverage[toml]  # It's pulled in by pytest-cov, but I want the toml ability.
 ipython
-jinja2
 numba
 pip-tools>=6.6.0
 pre-commit
-pycparser
```

Comment:
I'm guessing jinja2 and pycparser were from the days when katfgpu vendored spead2 and they were needed to bootstrap it?

Reply:
Yes. I had thought I'd written that already but looking back I see it was in the one commit message, not in the PR text.